### PR TITLE
Adds the ability to view saved posts when logged in.

### PR DESCRIPTION
**Pull request by Protuhj**
This works in concert with the 'Save' option available in the menu
option for posts, not comments.
Just view your profile in the app, then hit the options button and hit
'View saved posts'.

### CODE REVIEW

File: res/values-de/strings.xml [hunk 1]
```diff
@@ -109,6 +109,7 @@
 	<!--  === Menus === -->
 	
 	<string name="compose_message">Nachricht verfassen</string>
+	<string name="view_saved_posts">Gespeicherte Beiträge anzeigen</string>
```
Comment (zagaberoo) [inline res/values-de/strings.xml]: Same deal here, please revert changes to values-de/

File: src/in/shick/diode/threads/DownloadThreadsTask.java [hunk 2]
```diff
@@ -96,10 +113,12 @@ public DownloadThreadsTask(Context context, HttpClient client, ObjectMapper om,
         mOm = om;
         mSortByUrl = sortByUrl;
         mSortByUrlExtra = sortByUrlExtra;
-        if (subreddit != null)
+        mDTTSavedURI = null;
+        if (subreddit != null) {
             mSubreddit = subreddit;
-        else
+        } else {
```
Comment (zagaberoo) [inline src/in/shick/diode/threads/DownloadThreadsTask.java]: Great comments where helpful *and* cleaning up sloppy implicit braces, woohoo!

**Comment by zagaberoo** [comment]
Ready for me to roll out a new version, or should I hold off for anything?

File: src/in/shick/diode/threads/DownloadThreadsTask.java (hunk 2 above)
Comment (Protuhj) [reply src/in/shick/diode/threads/DownloadThreadsTask.java]: Sorry, I get anal about formatting.

**Comment by Protuhj** [comment]
That's up to you, I'm just merging nuggets out of what I wanted to do with my version, so they're not as far off.

File: src/in/shick/diode/threads/DownloadThreadsTask.java (hunk 2 above)
Comment (zagaberoo) [reply src/in/shick/diode/threads/DownloadThreadsTask.java]: Nothing to be sorry about, they're good changes.

**Pull request merged**